src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +15,16 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final User[] userService = UserService.getUsers();
```

Comment:
```suggestion
    private final UserService userService = ...;
```

Reply:
why we need to use exactly class UserService ? Without class User i can't use getters for email and password